terraform/nextstrain_batch.tf

```diff
@@ -0,0 +1,179 @@
+variable "nextstrain-batch-params" {
+  type  = map
+  default = {
+    retry_strategy = 1
```

Comment:
i don't know that we should retry at all.  most errors are resource limit issues, and i don't think we're going to resolve that with retries.

Reply:
these options were specified from the nextstrain setup docs as well, but i can take it out